SofaKernel/modules/SofaBaseLinearSolver/src/SofaBaseLinearSolver/CompressedRowSparseMatrix.h

```diff
@@ -212,8 +201,8 @@ public :
         }
         else
         {
-            msg_info_when(EMIT_EXTRA_MESSAGE)
-                    << ": resize("<<nbBRow<<"*"<<NL<<","<<nbBCol<<"*"<<NC<<")" ;
+            msg_info_when(SPARSEMATRIX_VERBOSE)
```

Comment:
A little pedantic but to more consistent with the other macro, should it not be COMPRESSEDROWSPARSEMATRIX_VERBOSE (or CRSMATRIX_VERBOSE) ?

Reply:
Makes sense. Will do :muscle:

Reply:
Done :wine_glass: